alerts/KubeAggregatedAPIErrors.md

```diff
@@ -0,0 +1,79 @@
+# KubeAggregatedAPIErrors
+
+## Meaning
+
+[This alert][KubeAggregatedAPIErrors] is triggered when multiple calls to the
+aggregated API of OpenShift fail over a certain period.
+
+## Impact
+
+Errors on the aggregated API can result in the unavailability of some OpenShift
+services.
+
+## Diagnosis
+
+The alert should contain information about the affected API and the scope of the
+impact.
+
+```text
+ - alertname = KubeAggregatedAPIErrors
+ - name = v1.packages.operators.coreos.com
+ - namespace = default
+...
+ - message = Kubernetes aggregated API v1.packages.operators.coreos.com/default has reported errors. It has appeared unavailable 5 times averaged over the past 10m.
+```
+
+## Mitigation
+
+### Check the APIs status checks are on True
+
+Currently, there are at least four aggregated APIs in an OpenShift Cluster. The
+API on the `openshift-apiserver` namespace, the prometheus-adapter on the
+namespace `openshift-monitoring`, the package-server service in the
+`openshift-operator-lifecycle-manager` namespace, and the API on the
+`openshift-oauth-apiserver` namespace. However, it makes sense to check the
+availability of all APIs.
+
+To get a list of `APIServices` and their backing aggregated APIs, use the
+following command:
+
+```console
+$ oc get apiservice
```

Comment:
it is better to remove `$`, otherwise it will copied to clipboard, and we need to remove it manually to run the command, same for others